lib/cWarnings.ml

```diff
@@ -176,3 +176,8 @@ let create ~name ~category ?(default=Enabled) pp =
                   str category ++ str "]"
               in
               Feedback.msg_warning ?loc msg
+
+type deprecation =
+  { since: string;
+    note: string;
+  }
```

Comment:
Really? As I mentioned, as soon as we have attributes, I'll extend to `Definition`, `Lemma` etc. I wanted to avoid doing another breaking change then, for clients of the API. But I'll do as you like, just let me know.

Reply:
I dunno, the current location seems confusing; and I guess we will see what happens when "attributes" arrive.